### PR TITLE
Updated to Swift 5.

### PR DESCRIPTION
**Pull request by kajensen**

### CODE REVIEW

File: Sources/iOS/NavigationDrawer/NavigationDrawerController.swift [hunk 1]
```diff
@@ -1218,6 +1218,8 @@ extension NavigationDrawerController: UIGestureRecognizerDelegate {
       }
       
     case .possible:break
+    @unknown default:
+      fatalError()
```
Comment (OrkhanAlikhanov) [inline Sources/iOS/NavigationDrawer/NavigationDrawerController.swift]: Material may only crash during development for the cases where developer does something disallowed like accessing `Application.shared` in an "iOS App Extension".

File: Sources/iOS/NavigationDrawer/NavigationDrawerController.swift [hunk 2]
```diff
@@ -1275,6 +1277,8 @@ extension NavigationDrawerController: UIGestureRecognizerDelegate {
...
```
Comment (OrkhanAlikhanov) [inline Sources/iOS/NavigationDrawer/NavigationDrawerController.swift]: Same as above

File: Sources/iOS/NavigationDrawer/NavigationDrawerController.swift (hunk 1 above)
Comment (OrkhanAlikhanov) [inline Sources/iOS/NavigationDrawer/NavigationDrawerController.swift]: Material may only crash during development for the cases where developer does something disallowed like accessing Application.shared in an "iOS App Extension".

File: Sources/iOS/NavigationDrawer/NavigationDrawerController.swift (hunk 2 above)
Comment (OrkhanAlikhanov) [inline Sources/iOS/NavigationDrawer/NavigationDrawerController.swift]: Same as above

**Comment by danieldahan** [comment]
@kajensen thank you very much for this. There are some conflicts and @OrkhanAlikhanov is correct, that the Swift updates, we prefer to do, as we need to understand the changes. Thank you so much for sharing this PR with us.

**Pull request closed without merging**